packages/tools/src/tools/base/ContourSegmentationBaseTool.ts

```diff
@@ -65,6 +65,12 @@ abstract class ContourSegmentationBaseTool extends ContourBaseTool {
   protected createAnnotation(
     evt: EventTypes.InteractionEventType
   ): ContourAnnotation {
+    if (this.configuration.interpolation?.enabled) {
+      InterpolationManager.addTool(this.getToolName());
+    } else {
```

Comment:
Shouldn't this be called on okComfiguration change or similar thing we have and i have forgotten?